README.md

```diff
@@ -71,10 +71,10 @@ See **Installation Instructions** for each available [release](https://github.co
 > | cardano-wallet | cardano-node (compatible versions) | SMASH (compatible versions)
 > | --- | --- | ---
 > | `master` branch | [1.25.1](https://github.com/input-output-hk/cardano-node/releases/tag/1.25.1) | [1.3.0](https://github.com/input-output-hk/smash/releases/tag/1.3.0)
+> | [v2021-03-04](https://github.com/input-output-hk/cardano-wallet/releases/tag/v2021-03-04) | [1.25.1](https://github.com/input-output-hk/cardano-node/releases/tag/1.25.1) | [1.3.0](https://github.com/input-output-hk/smash/releases/tag/1.3.0)
```

Comment:
I'm not sure we used to add the pending release at this stage, but the checklist says that now:

> Add an entry to the compatibility matrix in
README.md
for this release.
Check that versions match stack.yaml.
Keep info about the last 3 versions of cardano-wallet.

cc @piotr-iohk

Reply:
Yes, I modified the checklist order a couple releases ago. It's fine to do this earlier because the cabal versions are also bumped to a pending release version.